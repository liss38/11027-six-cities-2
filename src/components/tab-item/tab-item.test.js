import React from 'react';
import renderer from 'react-test-renderer';
import TabItem from './tab-item.jsx';

it(`Active 'TabItem' rendered correctly`, () => {
  const tree = renderer
    .create(<TabItem
      city={{
        id: 3,
        name: `Brussels`,
        location: {
          latitude: 52.370216,
          longitude: 4.895168,
          zoom: 10,
        },
      }}
      isActive={true}
      onCityClick={jest.fn()}
    />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});

it(`Not active 'TabItem' rendered correctly`, () => {
  const tree = renderer
    .create(<TabItem
      city={{
        id: 4,
        name: `Hamburg`,
        location: {
          latitude: 52.370216,
          longitude: 4.895168,
          zoom: 10,
        },
      }}
      isActive={false}
      onCityClick={jest.fn()}
    />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
