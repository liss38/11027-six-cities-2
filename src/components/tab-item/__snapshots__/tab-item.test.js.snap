// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Active 'TabItem' rendered correctly 1`] = `
<li
  className="locations__item"
>
  <a
    className="locations__item-link tabs__item tabs__item--active"
    href="#"
    onClick={[Function]}
  >
    <span>
      Brussels
    </span>
  </a>
</li>
`;

exports[`Not active 'TabItem' rendered correctly 1`] = `
<li
  className="locations__item"
>
  <a
    className="locations__item-link tabs__item "
    href="#"
    onClick={[Function]}
  >
    <span>
      Hamburg
    </span>
  </a>
</li>
`;
